desc/compute/_profiles.py

```diff
@@ -686,6 +744,27 @@ def _gradbeta_a(params, transforms, profiles, data, **kwargs):
     return data
 
 
+@register_compute_fun(
+    name="iota_23",
+    label="\\iota_{2/3}",
+    units="~",
+    units_long="None",
+    description="Rotational transform (normalized by 2pi) at rho=2/3",
+    dim=0,
+    params=[],
+    transforms={"grid": []},
+    profiles=[],
+    coordinates="",
+    data=["rho", "iota"],
+    method="str: Interpolation method. Default 'cubic'.",
+)
+def _iota_23(params, transforms, profiles, data, **kwargs):
```

Comment:
I'm not crazy about this as its own compute function. I feel like it would make more sense to have the power from iss04 as a compute function that uses iota and interpolates locally to get the 2/3 value